lib/runner/request-helpers-presend.js

```diff
@@ -159,6 +159,10 @@ module.exports = [
 
         if (!request) { return done(new Error('No request to send.')); }
 
+        // if body is disabled than skip loading files.
+        // @todo this may cause problem if body is enabled/disabled programmatically from pre-request script.
```

Comment:
Changes made to `pm.request` will not have any effect on the actual being sent (for now)